examples/readers/src/llamaparse-json.ts

```diff
@@ -0,0 +1,30 @@
+import fs from "fs/promises";
+import { LlamaParseReader } from "llamaindex";
+
+async function main() {
+  // Load PDF using LlamaParse json mode
+  const reader = new LlamaParseReader({ resultType: "json" });
+  const jsonObjs = await reader.loadJson("../data/uber_10q_march_2022.pdf");
```

Comment:
can we use an existing PDF?

Reply:
the file is straight from python llama_index, so I hoped licensing would be fine.
We can use an existing pdf, it's just that none of them have an interesting underlying structure with images, tables, etc.. to highlight LlamaParse json mode capabilities.